### DISCUSSION TITLE
Not passing optimizer to manual backward causes downstream error

### DISCUSSION THREAD

**Discussion by surya-narayanan** [Lightning-AI/pytorch-lightning#7620] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
![image](https://user-images.githubusercontent.com/17240858/118919797-117a0700-b8ea-11eb-90e4-5a9f057697c6.png)
It seems that if I don't pass in an optimizer to my manual backward, my gradient clipping passes through a None object. I thought passing an optimizer to manual backward was not required. It turns out, passing an optimizer doesn't solve the bug.

**Comment by carmocca** [comment, marked as answer]
Can you reproduce it and open an issue about it?

Discussions is more for help/questions
